pkg/retry/config.go

```diff
@@ -79,3 +80,76 @@ func newConfig(ctx context.Context, lu envconfig.Lookuper) (*Config, error) {
 	}
 	return &cfg, nil
 }
+
+// ToFlags binds the config to the give [cli.FlagSet] and returns it.
+func (cfg *Config) ToFlags(set *cli.FlagSet) *cli.FlagSet {
+	f := set.NewSection("COMMON SERVER OPTIONS")
+
+	f.StringVar(&cli.StringVar{
+		Name:   "github-app-id",
+		Target: &cfg.AppID,
+		EnvVar: "GITHUB_APP_ID",
+		Usage:  `The provisioned GitHub App reference.`,
+	})
+
+	f.StringVar(&cli.StringVar{
+		Name:   "big-query-project-id",
+		Target: &cfg.BigQueryProjectID,
+		EnvVar: "BIG_QUERY_PROJECT_ID",
+		Usage:  `The project ID where your BigQuery instance exists in.`,
+	})
+
+	f.StringVar(&cli.StringVar{
+		Name:   "bucket-url",
+		Target: &cfg.BucketURL,
+		EnvVar: "BUCKET_URL",
+		Usage:  `The URL for the bucket that holds the lock to enforce synchronous processing of the retry service.`,
+	})
+
+	f.StringVar(&cli.StringVar{
+		Name:   "checkpoint-table-id",
+		Target: &cfg.CheckpointTableID,
+		EnvVar: "CHECKPOINT_TABLE_ID",
+		Usage:  `The checkpoint table ID within the dataset.`,
+	})
+
+	f.StringVar(&cli.StringVar{
+		Name:   "dataset-id",
+		Target: &cfg.DatasetID,
+		EnvVar: "DATASET_ID",
+		Usage:  `The dataset ID within the BigQuery instance.`,
+	})
+
+	f.DurationVar(&cli.DurationVar{
+		Name:    "lock-ttl-clock-skew",
+		Target:  &cfg.LockTTLClockSkew,
+		EnvVar:  "LOCK_TTL_CLOCK_SKEW",
```

Comment:
Add gma prefix to such config variables?

Reply:
its a little redundant so i left it off for GMA.